app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1017,15 +1013,17 @@ fun PostListingList(
                     if (showCommunityName) {
                         CommunityLink(
                             community = postView.community,
-                            onClick = onCommunityClick,
+                            onClick = {},
+                            clickable = false,
                             showDefaultIcon = false,
                         )
                         DotSpacer(0.dp)
                     }
                     PersonProfileLink(
                         person = postView.creator,
                         isModerator = isModerator,
-                        onClick = onPersonClick,
+                        onClick = {},
```

Comment:
I don't think this is a good idea. Now users won't be able to go to a person's profile, or a community, from the post listing, at all. 

@twizmwazin thoughts?

Reply:
There was na issue opened about this, I'll link it here if I can find it. Users (including myself) accidentally fat-finger the small touch targets frequently and it is a bit of an annoyance. It is simple enough to open the post and then click the appropriate link from in there. An option wouldn't be a bad idea, which is what I think Boost does.

Reply:
Here's the issue: https://github.com/dessalines/jerboa/issues/488

Reply:
mmmk, I spose its fine, especially since this just affects the list version.